### PR TITLE
[MT-1177] Visitor Switching

### PR DESCRIPTION
**Pull request by Enricoza**

### CODE REVIEW

**Review by trister1997 — APPROVED** [review]

File: APIs/Tealium.Common/TealiumConfig.cs [hunk 1]
```diff
@@ -176,6 +191,8 @@ public sealed class TealiumConfig
         /// <param name="useRemoteLibrarySettings">Optional - enables/disbles fetching remote library settings</param>
         /// <param name="visitorServiceEnabled">Optional - enables/disables the visitor service</param>
         /// <param name="remoteCommands">Optional - collection of <see cref="IRemoteCommand"/> objects</param>
+        /// <param name="overrideConsentCategoriesKey">Optional - Overrides the consent categories key</param>
```
Comment (jameskeith) [inline APIs/Tealium.Common/TealiumConfig.cs]: Shouldn't this be the param doc for `visitorIdentityKey` ?

File: TealiumXamarinExample/TealiumXamarinExample/ViewModels/LoginViewModel.cs [hunk 2]
```diff
@@ -4,22 +4,70 @@
...
+        private void ClearStoredVisitorIds(object obj)
+        {
+            Helper.DefaultInstance.ClearStoredVisitorIds();
+        }
+
+        private void ResetVisitorId(object obj)
+        {
+            Helper.DefaultInstance.ResetVisitorId();
+        }
+
```
Comment (jameskeith) [inline TealiumXamarinExample/TealiumXamarinExample/ViewModels/LoginViewModel.cs:55]: whitespace is a bit much here and lines 69->71.

File: Bindings/android/bindings/build.gradle [hunk 3]
```diff
@@ -41,10 +41,10 @@ dependencies {
...
 
     embed "com.tealium:remotecommands:1.0.1"
-    embed "com.tealium:kotlin-core:1.4.1"
+    embed "com.tealium:kotlin-core:1.5.1"
     embed "com.tealium:kotlin-collect-dispatcher:1.1.0"
     embed "com.tealium:kotlin-lifecycle:1.1.1"
-    embed "com.tealium:kotlin-remotecommand-dispatcher:1.1.1"
-    embed "com.tealium:kotlin-tagmanagement-dispatcher:1.1.0"
-    embed "com.tealium:kotlin-visitor-service:1.1.0"
+    embed "com.tealium:kotlin-remotecommand-dispatcher:1.2.0"
```
Comment (jameskeith) [inline Bindings/android/bindings/build.gradle]: if it's a straigthforward switch, then `1.2.1` is the latest version of the RemoteCommandDispatcher

**Review by jameskeith — APPROVED** [review]
looks good to me

**Pull request merged**